core/src/core/block.rs

```diff
@@ -423,6 +423,22 @@ impl From<UntrustedBlockHeader> for BlockHeader {
 	}
 }
 
+/// List of block headers
+pub struct BlockHeaders(pub Vec<BlockHeader>);
+
+/// Deserialization of a list of block headers
+impl Readable for BlockHeaders {
```

Comment:
We used to "stream" headers in chunks of 32. 
`BlockHeaders` with a single `read` means this is no longer the case?